scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobRemoveHandler.java

```diff
@@ -61,64 +69,64 @@ private boolean isInFinishedState(InternalJob job) {
 
     @Override
     public Boolean call() {
+        boolean allJobsWereRemoved;
         try {
             long start = 0;
 
             if (logger.isInfoEnabled()) {
                 start = System.currentTimeMillis();
-                logger.info("Removing job " + jobId);
+                logger.info("Removing jobs " + jobIds.stream().map(JobId::value).collect(Collectors.joining(", ")));
             }
 
             SchedulerDBManager dbManager = service.getInfrastructure().getDBManager();
 
-            List<InternalJob> jobs = dbManager.loadJobWithTasksIfNotRemoved(jobId);
-            TerminationData terminationData;
+            List<InternalJob> jobs = dbManager.loadJobWithTasksIfNotRemoved(jobIds.toArray(new JobId[0]));
```

Comment:
Call me paranoid, but JobId length should be check here as well.

Reply:
YOU ARE PARANOID!!
(yeah i will check it as well)